editor/src/components/canvas/canvas-strategies/strategies/set-flex-gap-strategy.tsx

```diff
@@ -200,7 +197,7 @@ export const setFlexGapStrategy: CanvasStrategyFactory = (
             },
           ]),
         ],
-        [...selectedElements, ...children.map((c) => c.elementPath)],
+        selectedElements,
```

Comment:
do you remember why were the children included here to begin with? I'm afraid this was to fix a behavior.

Reply:
this line was taken from the original `setElementsToRerender` (see https://github.com/concrete-utopia/utopia/pull/6408), and it was added with the original implementation of the control (https://github.com/concrete-utopia/utopia/pull/2755).

We don't actually need to explicitly pass the children paths here, because they aren't affected by the strategy